Sources/ClientRuntime/Auth/HTTPAuthAPI/Signer.swift

```diff
@@ -0,0 +1,18 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+
+public protocol Signer {
+    associatedtype IdentityT: Identity
```

Comment:
Why call this type IdentityT? Its slightly confusing since T is usually reserved for when dealing with generics

Reply:
Swiftlint complains when `T` is used, saying generic name has to be at least 3 characters long. `IdentityT` is used to match generic name used in SRA.